client/blocks/app-promo/index.jsx

```diff
@@ -148,29 +153,36 @@ export class AppPromo extends Component {
 		const { translate } = this.props;
 		const buttons = [ { action: 'cancel', label: translate( 'OK' ) } ];
 
+		const message = displayJetpackAppBranding ? (
+			<p className="app-promo__paragraph">
+				<a href="https://wp.com/app" className="app-promo__jetpack-app-link">
+					Get the Jetpack app
+				</a>{ ' ' }
+				{ translate( 'to use Reader anywhere, any time.' ) }
+			</p>
+		) : (
+			<p className="app-promo__paragraph">
+				{ translate( 'WordPress.com in the palm of your hands — download the mobile app.' ) }
+			</p>
+		);
+
 		return (
 			<div className="app-promo">
 				<Button
-					tabIndex="0"
-					className="app-promo__dismiss"
-					onClick={ this.dismiss }
-					aria-label={ translate( 'Dismiss' ) }
-				>
-					<Gridicon icon="cross" size={ 24 } />
-				</Button>
-				<Button
-					onClick={ this.sendMagicLink }
-					className="app-promo__link"
+					onClick={ ! displayJetpackAppBranding && this.sendMagicLink }
+					className={ classNames( 'app-promo__link', {
+						'app-promo_jetpack-app-button': displayJetpackAppBranding,
+					} ) }
 					title="Try the mobile app!"
 				>
 					<img
 						className="app-promo__icon"
-						src={ wordpressLogoImage }
-						width="32"
-						height="32"
+						src={ displayJetpackAppBranding ? jetpackLogoImage : wordpressLogoImage }
+						width={ displayJetpackAppBranding ? '25' : '32' }
```

Comment:
Should the dimensions be numbers and not strings?

Reply:
It looks like either can be used, I've gone ahead to update to numbers in https://github.com/Automattic/wp-calypso/pull/68293/commits/6a8513039095417c302c6f4a901325cb7b9fa412 as that seems more 'proper'. :)